res/controllers/Denon-MC7000-scripts.js

```diff
@@ -89,13 +89,26 @@ MC7000.scratchParams = {
     beta: (1.0/10)/32
 };
 
-// Sensitivity factor of the jog wheel (also depends on audio latency)
-// 0.5 for half, 2 for double sensitivity - Recommendation:
-// set to 0.5 with audio buffer set to 50ms
-// set to 1 with audio buffer set to 25ms
-// set to 3 with audio buffer set to 5ms
-MC7000.jogSensitivity = 1;
-
+// Jog wheel parameters
+MC7000.jogParams = {
+    // Sensitivity factor of the jog wheel (also depends on audio latency)
+    // 0.5 for half, 2 for double sensitivity - Recommendation:
+    // set to 0.5 with audio buffer set to 50ms
+    // set to 1 with audio buffer set to 25ms
+    // set to 3 with audio buffer set to 5ms
+    sensitivity: engine.getSetting("jogSensitivity") || 1,
```

Comment:
I guess for the jog the input-latency matters. Because for the output-latency we have an undocumented CO: `engine.getValue("[App]", "output_latency_ms")`.